src/frontend/src/lib/components/ui/HeroButtonGroup.svelte

```diff
@@ -0,0 +1,12 @@
+<div
+	role="toolbar"
+	class="toolbar w-full px-4 py-6 text-black inline-flex justify-between rounded-[32px] bg-[var(--color-primary)]"
+>
+	<slot />
+</div>
+
+<style lang="scss">
+	.toolbar > :global(:last-child) {
```

Comment:
Is there another way? Tailwing has a built in `last-child` selector I think for example

Reply:
i tried that and it was not working, but I am don't remember why. Let me re-try

Reply:
So, as I remembered: `last:border-r-0` or  `last:border-0` or `last:border-none` do not work... not even `last:!border-0`

I am not really sure why, I cannot find any documentation about it, but I have to imagine that it has something to do with the cascading of CSS

any suggestion?

Reply:
ok... found the solution... i was being stupid... the class goes inside `ButtonHero` and not in the button group component